docs/src/examples/components/Menu/Usage/MenuExampleKind.shorthand.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { Menu } from '@stardust-ui/react'
+
+const items = [
+  { key: 'editorials', content: 'Editorials' },
+  { key: 'review', content: 'Reviews' },
+  { key: 'separator', kind: 'MenuSeparator' },
```

Comment:
As I remember we decided to use lowercase without prefixes in the `kind` prop, i.e. `separator`.
And technically, it's a `divider`